docs/draft/17-url-scheme.md

```diff
@@ -0,0 +1,42 @@
+---
+permalink: /spec/17
+parent: Draft specs
+title: 17/URL SCHEME
+---
+
+# 17/Defined Url Scheme
+
+> Version: 0.1.0
+>
+> Status: Draft
+>
+> Authors: Aleksandar Djenic <aleksandardjenic@status.im>
+>
+
+
+## Table of Contents
+
+ 1. [Url Scheme](#url-scheme)
+ 2. [Defined urls](#defined-urls)
+
+## Url scheme
+
+- internal: “status-im://”
+- external: “https://join.status.im”
```

Comment:
Why are you using [left](https://www.compart.com/en/unicode/U+201C) and [right](https://www.compart.com/en/unicode/U+201D) double quotes instead of [normal double quotes](https://www.compart.com/en/unicode/U+0022)?

Reply:
@jakubgs that's most likely case I have local languages added, and didn't pay attention when it switched. I fixed that part. Thanks.